src/Microsoft.ML.HalLearners/SymSgdClassificationTrainer.cs

```diff
@@ -833,7 +833,7 @@ private static unsafe class Native
                 }
             }
 
-            [DllImport(DllName), SuppressUnmanagedCodeSecurity]
+            [DllImport(NativePath, CallingConvention = CallingConvention.Cdecl), SuppressUnmanagedCodeSecurity]
```

Comment:
This is the only `Decl` in this C# file?

Reply:
not any more!

---
In reply to: [239658431](https://github.com/dotnet/machinelearning/pull/1833#discussion_r239658431) [](ancestors = 239658431)